a03_barkovskan.py

```diff
@@ -0,0 +1,115 @@
+# Author: Nadia Barkovska
+# Username: barkovskan
+#
+# Assignment: a03
+# Purpose:
+# Google Doc Link: https://docs.google.com/document/d/1-ibxsP64Y9TbDlPXR_dkQkF39-UAC4n8e4f66QF4NnQ/edit?usp=sharing
+#
+#################################################################################
+# Acknowledgements:
+#
+#
+#################################################################################
+
+import turtle
+
+
+def drawsquare (bubbles, size):
+    """" I defined function drawsquare to get turtle "bubbles" to draw a square """
+    for i in range(4):
+        bubbles.forward(size)
+        bubbles.left(90)
+    # ....
+
+
+def main ():
+    """I used the main function to define some attributes and invoke drawsquare function."""
+    wn = turtle.Screen()
+    wn.bgcolor("black")
+    bubbles = turtle.Turtle()
+    bubbles.speed(0)
+    bubbles.penup()
+    bubbles.setpos(-220, 100)
+    bubbles.pendown()
+
+    for i in range(8):
+        if (i) % 2 == 0:
+            bubbles.fillcolor("pink")
+        else:
+            bubbles.fillcolor(0.245,0.245,0.220)
+        bubbles.begin_fill()
+        drawsquare(bubbles, 60)
+        bubbles.forward(60)
+        bubbles.end_fill()
+
+
+    bubbles.penup()
```

Comment:
From here to the end of the main function, I can see multiple functions. Refactor your code (hint: highlight code you think makes a good function, then right-click, select "Refactor >> Extract Method..." and see what happens.